source/installation-guide/open-distro/distributed-deployment/unattended/index.rst

```diff
@@ -8,18 +8,169 @@
+
+.. _insitialize_elasticsearch_unattended:
+
+Initialize Elasticsearch Cluster
+--------------------------------
+
+Once Elasticsearch has been installed in all the nodes we need to initialize the OpenDistro cluster. This proccess only needs to be done in one node.
+
+#. Download the script. Skip this step if it is already on the node:
+
+    .. code-block:: console
+
+      # curl -so ~/wazuh_install.sh https://packages.wazuh.com/wazuh_install/4.2/wazuh_install.sh 
+
+#. Run the script
+
+    .. code-block:: console
+
+      # ./wazuh_install -s
+
+.. _install_kibana_unattended:
+
+Installing Wazuh server
+-----------------------
+
+#. Download the script. Skip this step if it is already on the node:
+
+    .. code-block:: console
+
+      # curl -so ~/wazuh_install.sh https://packages.wazuh.com/wazuh_install/4.2/wazuh_install.sh 
+
+#. Run the script:
+
+    .. code-block:: console
+
+      # ./wazuh_install -w <wazuh-server-node-name>
+
+    The following values must be replaced:
+
+      - ``wazuh-server-node-name``: Name of the instance. This name must be the same used in ``config.yml`` for the configuration setting, e.g. ``wazuh``. 
+
+      
+Installing Kibana
+-----------------
+
+.. note:: It is necessary to initialize the Elasticsearch cluster before this step.
+
+#. Download the script. Skip this step if it is already on the node:
+
+    .. code-block:: console
+
+      # curl -so ~/wazuh_install.sh https://packages.wazuh.com/wazuh_install/4.2/wazuh_install.sh 
+
+#. Run the script:
+
+    .. code-block:: console
+
+      # ./wazuh_install -k <kibana-node-name>
+
+    The following values must be replaced:
+
+      - ``kibana-node-name``: Name of the instance. This name must be the same used in ``config.yml`` for the configuration setting, e.g. ``kibana``. 
+
+#. Access the web interface: 
+
+    The user and password for the registration will show at the end of the script execution, aditionally all the passwords are stored in the ``passwords_file.yml`` file in ``configurations.tar``.
+
+    .. code-block:: none
+
+      URL: https://<kibana_ip>
+      user: admin
+      password: admin_password  
+  
+
+    Upon the first access to Kibana, the browser shows a warning message stating that the certificate was not issued by a trusted authority. An exception can be added in the advanced options of the web browser or,  for increased security, the ``root-ca.pem`` file previously generated can be imported to the certificate manager of the browser.  Alternatively, a certificate from a trusted authority can be configured. 
+
+    .. note:: If Kibana is accessed before installing the Wazuh server, the Wazuh Kibana plugin indicates that it cannot establish a connection with the Wazuh API. Proceed with the Wazuh server installation to remediate this.
+
+    .. note:: It is highly recommended to change the passwords of the Elasticsearch users' passwords. To perform this action, see the :ref:`Elasticsearch tuning <change_elastic_pass>` section.
```

Comment:
I think this note is not necessary, the unattended installer change the password by a randomly generated one.

Reply:
Done 681e125.